src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -278,6 +282,15 @@ public function configureOptions(OptionsResolver $resolver)
             'text',
             'choice',
         ));
+
+        $resolver->setNormalizer('html5', function (Options $options, $html5) {
```

Comment:
It needs rebase to use `setDeprecated()`, later the normalizer can throw the exception?